osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -380,7 +380,7 @@ private void handleTimingPoint(string line)
             addControlPoint(time, new LegacyDifficultyControlPoint(beatLength)
 #pragma warning restore 618
             {
-                SpeedMultiplier = speedMultiplier,
+                SliderVelocity = speedMultiplier,
```

Comment:
One thing that came out of the tests above is - for scrolling rulesets, should `speedMultiplier` not be stored to the `EffectControlPoint` instead to have the equivalent effect? An example of a map that this caused a failure on is https://osu.ppy.sh/beatmapsets/460422#fruits/1000293, for instance.

(Incidentally that also raises the issue of the precision of that number, as the effect point's `ScrollSpeedBindable`'s precision is 0.01, rather than the epsilon that the legacy point used to use, but that might not be super relevant.)